CDP4Grapher/ViewModels/GraphElementViewModel.cs

```diff
@@ -122,15 +147,48 @@ public string Category
         /// <summary>
         /// Instanciate a <see cref="GrapherViewModel"/> updating its property with the given <see cref="NestedElement"/> property
         /// </summary>
-        /// <param name="nestedElement"></param>
-        public GraphElementViewModel(NestedElement nestedElement)
+        /// <param name="nestedElement">The represented nested element</param>
+        /// <param name="actualOption">The option</param>
+        public GraphElementViewModel(NestedElement nestedElement, Option actualOption)
         {
             this.Thing = nestedElement;
-            this.NestedElementElement = (ElementBase)nestedElement.ElementUsage.LastOrDefault() ?? nestedElement.RootElement;
+            this.actualOption = actualOption;
+            this.NestedElementElement = (ElementBase) nestedElement.ElementUsage.LastOrDefault() ?? nestedElement.RootElement;
             this.RegisterSubscriptions();
             this.UpdateProperties();
         }
-        
+
+        /// <summary>
+        /// Computes the full path of the element
+        /// </summary>
+        /// <returns>The element path</returns>
+        private string ComputePath()
```

Comment:
This was requested by @MartinFijneman ? a path of a element is in principle its modelcode. Why append names together?

Reply:
Actually It's Sam's request: [Issue 485](https://github.com/RHEAGROUP/CDP4-IME-Community-Edition/issues/485#issuecomment-680807683)

Reply:
But with Names? In the comment it just says path, maybe modelcode was meant?

Reply:
indeed, the intent was "modelcode"